src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -248,7 +254,7 @@ export class LanguageClientManager {
             if (toolchainPath) {
                 // eslint-disable-next-line @typescript-eslint/naming-convention
                 sourcekit.options = {
-                    env: { ...process.env, SOURCEKIT_TOOLCHAIN_PATH: toolchainPath },
+                    env: { ...sourcekit.options?.env, SOURCEKIT_TOOLCHAIN_PATH: toolchainPath },
```

Comment:
Is the `?` after `options` required? You just set `source kit.options` further up

Reply:
We should use either `!` or `?` here, and they should work the same because it's guaranteed to be created (but cannot be recognized by TS).